pkg/ibmcsidriver/constants.go

```diff
@@ -128,6 +128,9 @@ const (
 
 	// MinimumSDPVolumeSizeInBytes ... This is minimum size require for sdp (acadia profile)
 	MinimumSDPVolumeSizeInBytes int64 = 1 * utils.GiB
+
+	// Throughput ...
+	Throughput = "throughput"
```

Comment:
Do we need to call throughput on CSI side and bandwidht on RIAAS side ? Do we need to keep it same ?

Reply:
@sameshai , actually I align with volumeattributeclass and what k8s talk about so it looks we need to have this mapping as k8s always talk about throughput

Reply:
ok